test/unit/plugins/lint-block-indentation-test.js

```diff
@@ -8,6 +8,12 @@ generateRuleTests({
   config: 2,
 
   good: [
+    '<div>\n' +
```

Comment:
I've been slowly working on changing from string concat, to using arrays to do the concatting via `.join('\n')`.  Would you mind tweaking these to be like this:

```js
  [
    '<div>',
    '  \\{{example}}',
    '</div>'
  ].join('\n'),
```